### DISCUSSION TITLE
Navigation bar turns white in Android app despite dark mode

### DISCUSSION THREAD

**Discussion by lemon-sl** [standardnotes/forum#2924] [General]
**Describe the bug**
My phone's navigation/gesture bar turns white when I use Standard Notes, even though the phone's in dark mode. Problem only shows in this app & persists in both light and dark appearances.

**To Reproduce**
1. Switch on phone's Dark mode.
2. Open app.

**Expected behavior**
Navigation/gesture bar stays dark.

**Screenshots**
![Screenshot_20221231-233418](https://user-images.githubusercontent.com/121726420/210150023-1a6b19c8-6c5d-4b06-a8ba-ea683294f545.png)

**Smartphone:**
 - Device: Xperia 5III
 - OS: Android 12
 - Version: v3.45.42

**Comment by effieeee** [comment]
Hi @lemon-sl, v3.45.42 is a fairly old version, do you still experience this on the latest app? (v3.130.2)

**Comment by lemon-sl** [comment]
Hi unfortunately the problem's still there after I've updated to the latest version
